config/FpConfig.h

```diff
@@ -394,11 +394,11 @@ typedef FwIndexType FwQueueSizeType;
 // OS configuration
 
 #ifndef FW_HANDLE_MAX_SIZE
-#define FW_HANDLE_MAX_SIZE 24  //!< Maximum size of a handle for OS resources (files, queues, locks, etc.)
+#define FW_HANDLE_MAX_SIZE 72  //!< Maximum size of a handle for OS resources (files, queues, locks, etc.)
```

Comment:
Should there be a handle size per type of resource? Posix mutexes need 72, where it seems files and tasks only needed 24

Reply:
Not yet, no.